lua/lib/lfospec.lua

```diff
@@ -0,0 +1,449 @@
+-- LFOs for general-purpose scripting
+-- @module lib.lfospec
```

Comment:
forgive me, why would this lib not just be called `lfo`? is there something in supercollider that suggests this should have `spec` added?